src/entrypoints/background.ts

```diff
@@ -1,4 +1,16 @@
 import { WebRequest } from 'wxt/browser'
+import { handleUserToken } from './utils/handleUserToken'
+
+const FOUR_DAYS_IN_MS = 4 * 24 * 60 * 60 * 1000
+
+const interval = setInterval(() => {
+  console.log('Checking user token...')
```

Comment:
Are several console.log deliberately left out?

Reply:
Yes, so I can see in production that all the steps are executed correctly. But perhaps I can remove some of them.